src/main/java/com/adomasda/Main.java

```diff
@@ -1,11 +1,13 @@
 package com.adomasda;
 
+import com.adomasda.Floors.*;
+
 public class Main {
     public static void main(String[] args) {
-        Floor floor1 = new Floor(new Configuration(1, "available", SomeEnum.FIRST_FLOOR));
-        Floor floor2 = new Floor(new Configuration(2, "available", SomeEnum.SECOND_FLOOR));
-        Floor floor3 = new Floor(new Configuration(3, "unavailable", SomeEnum.SECOND_FLOOR));
-        Floor floor4 = new Floor(new Configuration(4, "available", SomeEnum.SECOND_FLOOR));
+        Floor floor1 = new FirstFloor(new Configuration("available", SomeEnum.FIRST_FLOOR));
+        Floor floor2 = new SecondFloor(new Configuration("available", SomeEnum.SECOND_FLOOR));
+        Floor floor3 = new ThirdFloor(new Configuration("unavailable", SomeEnum.SECOND_FLOOR));
+        Floor floor4 = new ForthFloor(new Configuration("available", SomeEnum.SECOND_FLOOR));
 
         Elevator elevator = new Elevator(floor1, floor2, floor3, floor4);
```

Comment:
minor: logical confuse - the elevator cannot contain floors :-) will need to refactor later